external/loaders/loaders/mappers/_nudged/_nudged.py

```diff
@@ -171,22 +175,31 @@ def open_nudge_to_fine(
     return XarrayMapper(ds.rename(rename_vars))
 
 
+@register_mapper_function
 def open_nudge_to_fine_multiple_datasets(
-    urls: Sequence[str], names: Optional[Sequence[Hashable]] = None, **kwargs
-):
+    data_path: str,
```

Comment:
Please let me know if this signature will cause issues with argo integration.

Reply:
I tried to look around for experiments that might use this mapper function, but I don't know where all those experiment configs reside now, if anywhere.

Reply:
Maybe get @brianhenn, @AnnaKwa, or @spencerkclark to chime in on this.

Reply:
We talked about this and decided this can be fine as part of moving to use configuration yamls to specify the data path, instead of CLI arguments (which we also discussed was fine)